spec/jobs/state_file/send_reject_resolution_reminder_notification_job_spec.rb

```diff
@@ -110,7 +111,36 @@
               message: message,
               body_args: body_args).exactly(2).times
           end
+        end
+
+        context "when another intake exists with the same hashed SSN" do
```

Comment:
can we add a test case where they have the same ssn but don’t have an efile submission?